test/instrumentatios/extentions/http.test.ts

```diff
@@ -74,7 +74,7 @@ describe('Capturing HTTP Headers/Bodies', () => {
 
   app.get('/circular_test', (req: any, res: any) => {
     http
-      .request({ host: 'localhost', port: 8000, path: '/test_get' }, res2 => {
+      .request({ host: 'localhost', port: 12334, path: '/test_get' }, res2 => {
```

Comment:
I recommend having a fix port for all tests (plus gRPC)

Reply:
fixed